.github/workflows/mvn-verify-check.yml

```diff
@@ -246,8 +251,57 @@ jobs:
           }
           done
 
+  verify-213-modules:
+    needs: set-scala213-versions
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJSON(needs.set-scala213-versions.outputs.sparkJDK17Versions) }}
+    steps:
+      - uses: actions/checkout@v4 # refs/pull/:prNumber/merge
+
+      - name: Setup Java and Maven Env
+        uses: actions/setup-java@v4
+        with:
+          distribution: adopt
+          java-version: 17
+
+      - name: check runtime before tests
+        run: |
+          env | grep JAVA
+          java -version && mvn --version && echo "ENV JAVA_HOME: $JAVA_HOME, PATH: $PATH"
+
+      - name: Build JDK
+        run: |
+          # https://github.com/NVIDIA/spark-rapids/issues/8847
+          # specify expected versions
+          export JAVA_HOME=${JAVA_HOME_${{ matrix.java-version }}_X64}
+          export PATH=${JAVA_HOME}/bin:${PATH}
+          java -version && mvn --version && echo "ENV JAVA_HOME: $JAVA_HOME, PATH: $PATH"
+           # verify Scala 2.13 build files
+          ./build/make-scala-version-build-files.sh 2.13
+          # verify git status
+          if [ -n "$(echo -n $(git status -s | grep 'scala2.13'))" ]; then
+              git add -N scala2.13/* && git diff 'scala2.13/*'
+              echo "Generated Scala 2.13 build files don't match what's in repository"
+              exit 1
+          fi
```

Comment:
We keep replicating this code over and over. 

We already have an `if_modified_files` execution. We need to make add an execution to the tune of  update_scala213_poms as we do with Conf generation `update_config_docs`, and drop all calls `./build/make-scala-version-build-files.sh 2.13` outside Maven

Reply:
I can create an issue for it to handle it as a follow-on

Reply:
https://github.com/NVIDIA/spark-rapids/issues/11157

Reply:
Yes